source/sink-connector/configuration-properties/write-strategies.txt

```diff
@@ -15,13 +15,43 @@ Overview
 
 .. _sink-configuration-write-model-strategy-description-start:
 
-Use the strategies in the following table to specify how the {+sink-connector+}
-writes data into MongoDB. You can specify a write strategy with the following
+You can set configuration properties to specify how the
+{+sink-connector+} writes data into MongoDB. The following sections
+describe the configuration properties that you can set to customize this
+behavior.
+
+Write Model
+~~~~~~~~~~~
+
+Set the ``writemodel.strategy`` configuration property to specify how
+the sink connector writes data when it receives a sink record.
+
+You can set the value of ``writemodel.strategy`` to any of the
+fully qualified class names of the write model strategies described in
+the :ref:`Strategies <sink-connector-write-model-strategies>` section of
+this page. You can specify a strategy by setting the following
+configuration:
+
+.. code-block:: properties
+
+   writemodel.strategy=<a write model strategy>
+
+Delete Write Model
+~~~~~~~~~~~~~~~~~~
+
+Set the ``delete.writemodel.strategy`` configuration property to specify
+how the sink connector writes data when it receives a tombstone
+event. The connector receives tombstone events when records are deleted.
```

Comment:
Nit: 

I would clarify the sentence: 
> "The connector receives tombstone events when records are deleted."

A tombstone message is a kafka record that contains a key but no value, and signifies a deleted record.